timescaledb/quick-start/python.md

```diff
@@ -72,36 +64,40 @@ CONNECTION = "dbname =tsdb user=tsdbadmin password=secret host=host.com port=543
 The above method of composing a connection string is for test or development purposes only, for production purposes be sure to make sensitive details like your password, hostname, and port number environment variables.
 </highlight>
 
-### Step 3: Connect to Timescale database using the Psycopg2 connect function
+### Step 3: Connect to TimescaleDB using the psycopg2 connect function
 
-We'll use the psycopg2 [connect function][psycopg2-connect] to create a new database session
+Use the psycopg2 [connect function][psycopg2-connect] to create a new database session and create 
+a new [cursor object][psycopg2-cursor] to interact with the database.
 
 In your `main` function, add the following lines:
 
 ```python
+CONNECTION = "postgres://username:password@host:port/dbname"
 def main():
-…
-	with psycopg2.connect(CONNECTION) as conn:
-		#Call the function that needs the database connection
-		func_1(conn)
+    with psycopg2.connect(CONNECTION) as conn:
+		cursor = conn.cursor()
+        # use the cursor to interact with your database
+        # cursor.execute("SELECT * FROM table")
 ```
 
-Alternatively, you can create a connection object as follows and the pass that object
+Alternatively, you can create a connection object and pass the object
 around as needed, like opening a cursor to perform database operations:
 
 ```python
-def main():
-conn = psycopg2.connect(CONNECTION)
-insert_data(conn)
-cur = conn.cursor()
+CONNECTION = "postgres://username:password@host:port/dbname"
+def main():  
+    conn = psycopg2.connect(CONNECTION)
+    cursor = conn.cursor()
+    # use the cursor to interact with your database
+    # cursor.execute("SELECT * FROM table")
```

Comment:
Maybe as the person is starting we can handle something like:
```suggestion
    print(cursor.execute("SELECT 'hello world'"))
```

Reply:
https://github.com/timescale/docs/pull/90/commits/981de17fa02761988313a37cbfba8f34670057e6